web/src/components/Chat.tsx

```diff
@@ -97,7 +121,11 @@ const Chat = ({
 
   return (
     <Container justifyContent='space-between'>
-      <ChatConversation messages={messages} isTyping={isTyping} />
+      {isLoading ? (
+        renderChatConversationSkeleton()
+      ) : (
+        <ChatConversation messages={messages} isTyping={isTyping} loading={isLoading} />
+      )}
```

Comment:
🤔 Since we pass `loading` anyway to `ChatConversation`, I think it would be better to also move the rendering to the skeleton to `ChatConversation`. That way, there is a clearer separation of concerns and less complexity here and you can perhaps reuse some of the components to get the size/widths for the skeleton.